connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorValidationIntegrationTest.java

```diff
@@ -460,7 +460,7 @@ public static abstract class AbstractTestConverter extends TestConverter {
     }
 
     public static class TestConverterWithPrivateConstructor extends TestConverter {
-        private TestConverterWithPrivateConstructor() {
+        public TestConverterWithPrivateConstructor() {
```

Comment:
I don't dig in it yet, but it seems the private constructor is what it tries to test ( according to the naming "TestConverterWithPrivateConstructor")